examples/basic/apps/docs/eslint.config.js

```diff
@@ -0,0 +1,5 @@
+import { config } from "@repo/eslint-config/next-js";
+import nextPlugin from "@next/eslint-plugin-next";
+
+/** @type {import("eslint").Linter.Config} */
+export default [...config];
```

Comment:
I left these as a spread operator so users can easily add more configurations if they want to. We can drop it to `export default config` if we want to.

Reply:
dropping it until we need to makes the most sense to me, but up to you!